snapshot/snapshot.go

```diff
@@ -116,8 +116,8 @@ func NewSnapshotter(ctx context.Context, cfg *config.Config) (snapshots.Snapshot
 		}
 	}()
 
-	if cfg.APISocket != "" {
-		systemController, err := system.NewSystemController(manager, cfg.APISocket)
+	if cfg.EnableSystemController {
+		systemController, err := system.NewSystemController(manager, path.Join(cfg.RootDir, "system_controller.sock"))
```

Comment:
Maybe name it `supervisor.sock` ?

Reply:
Um. We already have supervisors for nydusd daemon.  Or just rename it to system.sock ?snapshotter.sock ?

Reply:
`system.sock` is ok.

Reply:
updated